R/utils.R

```diff
@@ -7,10 +7,14 @@ is.fire <- function(x) inherits(x, "Fire")
 
 # dependencies
 is.dependency <- function(x) inherits(x, "dash_dependency")
-is.output <- function(x) is.dependency(x) && inherits(x, "output")
-is.input <- function(x) is.dependency(x) && inherits(x, "input")
-is.state <- function(x) is.dependency(x) && inherits(x, "state")
-is.event <- function(x) is.dependency(x) && inherits(x, "event")
+#is.output <- function(x) is.dependency(x) && inherits(x, "output")
```

Comment:
🔪  - in general, let's delete code instead of commenting it out. It'll always be available in git if we want to refer to it later

Reply:
Right, sorry about this; I had commented out these lines and others to test, will :hocho:.